code/modules/power/turbine/turbine.dm

```diff
@@ -233,6 +233,7 @@
 
 	/// The rotor this inlet is linked to
 	var/obj/machinery/power/turbine/core_rotor/rotor
+
```

Comment:
Like not needed a new line

Reply:
Sorry, didn't have the chance to go through my current changes just yet